compiler/luci/service/src/CircleShapeInferenceHelper.h

```diff
@@ -49,8 +49,8 @@ loco::TensorShape circle_shape(const luci::CircleNode *node);
 loco::TensorShape broadcast_shape(const loco::TensorShape &x, const loco::TensorShape &y);
 
 // Return shape of pad ops using paddings.
-loco::TensorShape pad_shape(const loco::TensorShape &input_shape,
-                            const luci::CircleConst *paddings);
+// If paddings is static, return the shape filled with unknown dimensions.
```

Comment:
Q) what if paddings is not static? does it return shape filled with known dimensions?

Reply:
Oh, It's typo... 😢
It return shape filled with unknown dimensions if paddings is **not** static